chall01/hhuhtane.py

```diff
@@ -0,0 +1,37 @@
+# **************************************************************************** #
+#                                                                              #
+#                                                         :::      ::::::::    #
+#    hhuhtane.py                                        :+:      :+:    :+:    #
+#                                                     +:+ +:+         +:+      #
+#    By: hhuhtane <hhuhtane@student.hive.fi>        +#+  +:+       +#+         #
+#                                                 +#+#+#+#+#+   +#+            #
+#    Created: 2020/03/26 15:47:35 by hhuhtane          #+#    #+#              #
+#    Updated: 2020/03/26 16:51:09 by hhuhtane         ###   ########.fr        #
+#                                                                              #
+# **************************************************************************** #
```

Comment:
Don't put the 42 header for those challenges, please

It is missing here the shebang to make your python script executable:

https://stackoverflow.com/questions/7670303/purpose-of-usr-bin-python3/7670338